apps/design/backend/scripts/db_reset_dev.sh

```diff
@@ -17,6 +17,6 @@ sudo systemctl start postgresql
 sudo -u postgres psql -c "drop database design;"
 sudo -u postgres psql -c "drop user design;"
 
-sudo -u postgres psql -c "create user design password 'design';" &&
+sudo -u postgres psql -c "create user design superuser password 'design';" &&
```

Comment:
Fine to unblock but in the long term I think it's preferable to apply more granular permissions.